mock/verify-mocks-compatibility.ts

```diff
@@ -0,0 +1,16 @@
+import * as RawMocks from ".";
```

Comment:
The only purpose of this file is to verify that mocks in `jest-mocks` and `index.ts` are in sync (i.e. same functions are exposed)

Reply:
I tried to remove a fn from the jest implementation, and it's actually not reported by TS, so I removed it.